README.md

```diff
@@ -11,7 +11,11 @@ An exporter for [Amazon Cloudwatch](http://aws.amazon.com/cloudwatch/), for Prom
 
 ## Credentials and permissions
 
-The Cloudwatch Exporter uses the [AWS Java SDK](http://docs.aws.amazon.com/AWSSdkDocsJava/latest/DeveloperGuide/welcome.html), which offers [a variety of ways to provide credentials](http://docs.aws.amazon.com/AWSSdkDocsJava/latest/DeveloperGuide/credentials.html). This includes the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` enviroment variables
+The Cloudwatch Exporter uses the
```

Comment:
HAProxy is mostly spelled "HAProxy" by HAProxy: http://www.haproxy.org/

In any case, we don't have to spend extraordinary amounts of effort to
always use a particularly sanctioned spelling, but if it's easy enough to
do so (like now, when we have the occasion), we should.

On Wed, Jan 21, 2015 at 8:10 PM, Brian Brazil <notifications@github.com>
wrote:

> In README.md
> <https://github.com/prometheus/cloudwatch_exporter/pull/1#discussion_r23324962>
> :
>
> > @@ -11,7 +11,11 @@ An exporter for [Amazon Cloudwatch](http://aws.amazon.com/cloudwatch/), for Prom
> >
> >  ## Credentials and permissions
> >
> > -The Cloudwatch Exporter uses the [AWS Java SDK](http://docs.aws.amazon.com/AWSSdkDocsJava/latest/DeveloperGuide/welcome.html), which offers [a variety of ways to provide credentials](http://docs.aws.amazon.com/AWSSdkDocsJava/latest/DeveloperGuide/credentials.html). This includes the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` enviroment variables
> > +The Cloudwatch Exporter uses the
>
> That affects other things too, we've other exporters that have non-obvious
> capitalisation to those not used to them such as HAProxy.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/prometheus/cloudwatch_exporter/pull/1/files#r23324962>
> .
>